packages/insomnia-app/app/ui/components/settings/general.js

```diff
@@ -390,6 +391,23 @@ class General extends React.PureComponent<Props, State> {
 
         <hr className="pad-top" />
 
+        <h2>Security</h2>
+        <div className="form-row pad-top-sm">
+          {this.renderBooleanSetting(
+            'Clear OAuth 2 session on start',
+            'clearOAuth2SessionOnRestart',
+            'Clears the session of the OAuth2 popup window every time insomnia is launched',
```

Comment:
Minor nitpick, but Insomnia should probably be capitalized
```suggestion
            'Clears the session of the OAuth2 popup window every time Insomnia is launched',
```